src/Sylius/Behat/Page/Admin/Shipment/IndexPage.php

```diff
@@ -63,6 +63,12 @@ public function showOrderPageForNthShipment(int $position): void
         $this->getOrderLinkForRow($position)->clickLink('#');
     }
 
+    public function shipShipmentOfNumberWithTrackingCode(string $orderNumber, string $trackingCode): void
+    {
+        $this->getField($orderNumber, 'actions')->fillField('Tracking code', $trackingCode);
```

Comment:
You could assign `$this->getField($orderNumber, 'actions')` to the variable as you are using it twice